src/index.js

```diff
@@ -31,7 +31,7 @@ async function main () {
   const command = core.getInput('command')
   const noPublish = core.getInput('noPublish')
   const key = core.getInput('key')
-  const scopes = core.getInput('scopes')
+  const scopes = core.getInput('scopes') || undefined
```

Comment:
|| undefined not needed, if it is undefined it already is undefined
This might be a better use for nullish coalescing
```
const foo = null ?? 'default string';
console.log(foo);
// Expected output: "default string"

const baz = 0 ?? 42;
console.log(baz);
// Expected output: 0
```

Reply:
But we want to set `scopes` to undefined when `core.getInput()` returns an empty string so that AJV ignores it when it gets to validation. If we do nullish coalescing, it will theoretically stay an empty string right?

```
const foo = '' ?? 'default string';
console.log(foo);
// Expected output: ''
```